javascript-bank/bank.js

```diff
@@ -0,0 +1,38 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (balance > 0 && Number.isInteger(balance)) {
+    var account = new Account(this.nextAccountNumber, holder);
+    account.deposit(balance);
+    this.accounts.push(account);
+    this.nextAccountNumber++;
+    return account.number;
+  } else {
+    return null;
+  }
+};
+
+Bank.prototype.getAccount = function (number) {
+  for (let i = 0; i < this.accounts.length; i++) {
+    if (this.accounts[i].number === number) {
+      return this.accounts[i];
+    }
+  }
+  return null;
+};
+
+Bank.prototype.getTotalAssets = function () {
+  if (this.accounts.length === 0) {
+    return 0;
```

Comment:
This conditional isn't necessary. If there is no length to the array, the loop would never start, so you're left with the same result.